### DISCUSSION TITLE
multiple <h1> header automatically added to every post

### DISCUSSION THREAD

**Discussion by qureshiaquib** [cotes2020/jekyll-theme-chirpy#1834] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi Team, I've found that my every post has multiple `<h1>` header by default added. because of which Bing indexer has highlighted this as critical
this seems to be added automatically by jekyll. is there a way we can remove one? I use TOC in every post is this because of this? i don't want to disable this feature.

`<h1 class="site-title"> <a href="/">Azure Doctor</a></h1>`
`<h1 data-toc-skip>Access Azure File Share from Remote sites via Entra ID joined devices</h1>`

### Operations you have already tried

NA, needed suggestions

**Comment by kungfux** [comment]
The thing is that these two are from different content categories. While the site title `<h1 class="site-title">` is under the sidebar, and the other `<h1 data-toc-skip="">` is under the article, I would say that this case may be handled incorrectly by Bing according to HTML5.

Anyway, to workaround this, if you really want to, you have to override [_includes/sidebar.html](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/_includes/sidebar.html#L15) to avoid using `<h1>` for the site title.

**Reply by qureshiaquib** [reply]
thanks Alex, so can i just remove this line of code? and i hope this won't impact the look and feel of the main blog name?

```
  <h1 class="site-title">
      <a href="{{ '/' | relative_url }}">{{ site.title }}</a>
    </h1>
```

**Reply by kungfux** [reply, marked as answer]
It will have impact. What you can do is to replace `h1` by `h2` or `p` and adjust styles to still look as `h1` site title on the left panel.

**Reply by qureshiaquib** [reply]
thanks Alex. I'll take it forward from here
